extensions/machine-learning-services/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "machineLearningServices",
+  "name": "machine-learning-services",
```

Comment:
please check that you don't have to change anywhere else - in build or excluded ext. or so?

Reply:
yeah good point. will chck

---
In reply to: [393905206](https://github.com/microsoft/azuredatastudio/pull/9519#discussion_r393905206) [](ancestors = 393905206)

Reply:
verified not used anywhere 

---
In reply to: [393910789](https://github.com/microsoft/azuredatastudio/pull/9519#discussion_r393910789) [](ancestors = 393910789,393905206)